src/main/java/org/sagebionetworks/web/client/widget/entity/MarkdownEditorWidget.java

```diff
@@ -148,9 +154,65 @@ public void configure(WikiPage page) {
 		view.setTitleEditorVisible(currentPage.getParentWikiId() != null);
 		view.setTitle(currentPage.getTitle());
 		globalApplicationState.setIsEditing(true);
+		setMarkdownTextAreaHandlers();
+		resizeMarkdownTextArea();
+		view.setDeleteClickHandler(getDeleteClickHandler());
 		view.showEditorModal();
 	}
 	
+	private void setMarkdownTextAreaHandlers() {
+		view.addTextAreaKeyUpHandler(new KeyUpHandler() {
+			@Override
+			public void onKeyUp(KeyUpEvent event) {
+				resizeMarkdownTextArea();
+			}
+		});
+		view.addTextAreaKeyUpHandler(new KeyUpHandler() {
+			@Override
+			public void onKeyUp(KeyUpEvent event) {
+				markdownEditorClicked();
+			}
+		});
+		view.addTextAreaClickHandler(new ClickHandler() {
+			@Override
+			public void onClick(ClickEvent event) {
+				markdownEditorClicked();
+			}
+		});
+	}
+	
+	private ClickHandler getDeleteClickHandler() {
+		return new ClickHandler() {
+			@Override
+			public void onClick(ClickEvent event) {
+				Bootbox.confirm(DisplayConstants.PROMPT_SURE_DELETE + " Page and Subpages?", new ConfirmCallback() {

```

Comment:
Need to do more if we want to test this, as we can't refer to bootbox (ui element) in a presenter test (simply moving this code to the presenter does not make it well tested).  could ask the view to display bootbox and hand it the confirm callback.  Then a couple of tests should be added to exercise this (where confirm is true, and another with confirm as false)